PokeAlarm/Utils.py

```diff
@@ -437,6 +437,14 @@ def get_time_as_str(t, timezone=None):
     return time_left, time_12, time_24
 
 
+# Return the time in seconds
+def get_time_in_seconds(t, timezone=None):
```

Comment:
This should probably be called something like 'get_seconds_remaining' to make it more clear what it does.

It doesn't need timezone, so we should remove it.

Reply:
Ok, i'll make that change.